### ISSUE TITLE
Support?

### ISSUE THREAD

**Issue by quqkuk** [45Drives/autotier#68]
Is this project still supported? There hasn't been a commit for 2 years, so I'm asking to understand whether this is still developed and used

**Comment by TheLinuxGuy** [comment]
I wouldn't rely on this at all. Use mergerfs instead and rsync.

or wait for bcachefs

**Comment by srzsh** [comment]
Yeah one can already use bcache or LVM's cache layer but I kept on stumbling on this repo and I was curious
